apps/mobile/src/services/notifications/NotificationService.ts

```diff
@@ -87,6 +105,18 @@ class NotificationsService {
     return isAuthorized
   }
 
+  async openDeviceSettings() {
+    try {
+      if (Platform.OS === 'ios') {
+        Linking.openSettings()
+      } else {
+        notifee.openNotificationSettings()
```

Comment:
why is it necessary that we use the notifee openNotificationSettings? isn't Linking.openSettings() also working on android?